### ISSUE TITLE
Add tracing configuration

### ISSUE THREAD

**Issue by objectiser** [nicolaferraro/integration-operator#1]
Would be great if an OpenTracing configuration could be provided in the CRD.

**Comment by objectiser** [comment]
Does not actually need to be in the same CRD, as it may better the separate the configuration of the runtime (including tracing config) from the route definition. So possibly a separate CRD for configuration that can be referenced in the `routes` CRD.

**Comment by nicolaferraro** [comment]
Yes, @objectiser tracing is really important, especially given the transient nature of containers in this proposal. They can be created and destroyed dynamically so, without tracing, we may not be able to see what's happening.

It should be possible to specify a global config at operator level and specific config in the CRD. At least a feature flag to enable/disable it as basic config.

Another option is to use a global config plus annotations on the CR for the local configuration. If the local configuration is not complex, it may be a nicer approach.
